homeassistant/components/dsmr/sensor.py

```diff
@@ -81,181 +80,181 @@ class DSMRSensorEntityDescription(SensorEntityDescription):
 SENSORS: tuple[DSMRSensorEntityDescription, ...] = (
     DSMRSensorEntityDescription(
         key="timestamp",
-        obis_reference=obis_references.P1_MESSAGE_TIMESTAMP,
+        obis_reference="P1_MESSAGE_TIMESTAMP",
```

Comment:
Instead of these being property string we use with getattr, we need to add lambda methods to the entity descriptions that fetches the right property direct. This will avoid the use of getattr.